blog/electron-36-0.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Electron 36.0.0
+date: 2025-04-28T00:00:00.000Z
+authors:
+  - georgexu99
+slug: electron-36-0
+tags: [release]
+---
+
+Electron 36.0.0 has been released! It includes upgrades to Chromium 136, V8 13.6, and Node 22.14.0.
+
+---
+
+The Electron team is excited to announce the release of Electron 36.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Bluesky](https://bsky.app/profile/electronjs.org) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Native macOS Context Menu Support
+
+Electron 36 enhances the `menu.popup` API by extending the `options` object to include `frame`. This feature enables integration with macOS system-level features like Writing Tools (spelling and grammar), Autofill, and Services menu items.
+
+```ts
```

Comment:
I think we usually add CommonJS JavaScript code in our blog post snippets (and throughout our docs). Is there a specific reason we switched to TS here?